homeworks/Ivan.Artemenko_IvanArtemenko/homework_5/homework_5_3.js

```diff
@@ -0,0 +1,9 @@
+function innerCopy(obj) {
+    const newObj = {};
+    Object.keys(obj).forEach((key) => { newObj[key] = obj[key]; });
+    return newObj;
+}
```

Comment:
this is not a deep copy operation, `objCopy.d` still references to `severalObj.d`